ros_buildfarm/templates/release/release_script.sh.em

```diff
@@ -47,8 +47,13 @@ cd $WORKSPACE
 
 echo ""
 echo "Get artifacts from source job"
-mkdir -p $WORKSPACE/binarydeb
-(set -x; cp $BASEPATH/source/sourcedeb/*.debian.tar.[gx]z $BASEPATH/source/sourcedeb/*.dsc $BASEPATH/source/sourcedeb/*.orig.tar.gz $WORKSPACE/binarydeb/)
+if [ -d "$BASEPATH/source/sourcedeb" ]; then
```

Comment:
Is this command checking for a Debian distribution to skip the steps in the case of rhel or any other RPM distribution? If that was the intention, I think will work better to check for exactly the type of distribution if possible.

Reply:
When I introduced RPM support in `ros_buildfarm`, I made a substantial effort to use terms like `sourcepkg` and `binarypkg` over terms like `sourcedeb` and `binarydeb`. However, there are a lot of places where it would simply be too much work to update the existing deb-specific code to stop using the more specific term in favor of the more generalized one.

This is one of those spots.

So in most places, instead of seeing a big mapping of os-to-specific-name that needs to be updated to support a new platform, you'll see a special (legacy) conditional for deb, and everything else (right now, only RPMs) uses the generalized term.

Clearly, however, this is even yet insufficient, because the copy operation I'm trying to add here is RPM-specific.

I'm open to suggestions on how to improve this.

Reply:
I was thinking on using the non-standard but useful ways to detect major Debian/RedHat distributions including their derivatives.  Check for `/etc/redhat-release` or `/etc/debian_version` could be a starting point. Might be adding to the current check?

Reply:
I took a different approach to solving this issue that puts the conditional in the template instead, which comes through significantly cleaner.